openslides_backend/shared/patterns.py

```diff
@@ -32,9 +32,10 @@ def __repr__(self) -> str:
         return f"Collection({repr(str(self))})"
 
     def __eq__(self, other: object) -> bool:
-        if not isinstance(other, Collection):
-            return NotImplemented
-        return self.collection == other.collection
+        try:
+            return self.collection == cast(Collection, other).collection
+        except Exception as e:
+            raise NotImplementedError(e)
```

Comment:
Maybe we should just return `False` here? There should be no reason to not be able to compare on of these classes with some random object, they are just not equal then.

Reply:
IMO we should insist on comparable objects. If we will change the type from FqId to string it will be useful to see the type.

Reply:
ok